src/System.Runtime/tests/System/ArgumentExceptionTests.cs

```diff
@@ -0,0 +1,66 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Xunit;
+using System.Tests;
```

Comment:
These `using System.Tests;`s shouldn't be necessary when all of the tests are in `namespace System.Tests;`.